utils/release_helper.py

```diff
@@ -0,0 +1,790 @@
+
+def update_milestone_in_issues(repo, issues_list) -> None:
+    outdate_issues = filter_outdated_items(repo, issues_list, "Issue")
+    if outdate_issues:
+        update_milestone(outdate_issues)
+
+
+def update_milestone_in_prs(repo, prs_list) -> None:
+    outdate_prs = filter_outdated_items(repo, prs_list, 'PR')
+    if outdate_prs:
+        update_milestone(outdate_prs)
+
+
+# Repo Stats
+def collect_release_info(repo) -> dict:
+    data = dict()
+    latest_release = get_latest_release(repo)
+    data["latest_release_created"] = latest_release.created_at
+    data["latest_release_published"] = latest_release.published_at
+    data["latest_release_url"] = latest_release.html_url
+    data["latest_version"] = get_latest_version(repo)
+
+    next_release_date = get_next_release_date(latest_release.published_at)
+    next_release_remaining_days = next_release_date - datetime.today()
+    data["next_release_date"] = next_release_date
+    data["next_release_remaining_days"] = next_release_remaining_days.days
+    data["next_release_state"] = get_next_release_state(repo)
+    data["next_release_version"] = get_next_release_version(repo)
+    next_stabilization_date = get_next_stabilization_date(next_release_date)
+    next_stabilization_remaining_days = next_stabilization_date - datetime.today().date()
+    data["next_stabilization_date"] = next_stabilization_date
+    data["next_stabilization_remaining_days"] = next_stabilization_remaining_days.days
+
+    previous_milestone = get_version_milestone(repo, data["latest_version"])
+    data["previous_milestone"] = previous_milestone.title
+    data["previous_milestone_closed_issues"] = previous_milestone.closed_issues
+    data["previous_milestone_open_issues"] = previous_milestone.open_issues
+    data["previous_milestone_total_issues"] = data["previous_milestone_closed_issues"]\
+        + data["previous_milestone_open_issues"]
+
+    active_milestone = get_latest_version_milestone(repo)
+    data["active_milestone"] = active_milestone.title
+    data["active_milestone_closed_issues"] = active_milestone.closed_issues
+    data["active_milestone_open_issues"] = active_milestone.open_issues
+    data["active_milestone_total_issues"] = data["active_milestone_closed_issues"]\
+        + data["active_milestone_open_issues"]
+    return data
+
+
+def print_specific_stat(status, current, total) -> None:
+    if current > 0:
+        percent = round((current / total) * 100.00, 2)
+        print(f'{status:21} {current:6} / {total:3} = {percent:4}%')
+
+
+def print_last_release_stats(data):
+    version = data['latest_version']
+    created_at = data['latest_release_created']
+    published_at = data['latest_release_published']
+    url_download = data['latest_release_url']
+    closed_issues = data["previous_milestone_closed_issues"]
+    total_issues = data["previous_milestone_total_issues"]
+
+    print('Information based on the latest published release:')
+    print(f'Current Version:\t {version}\n'
```

Comment:
Regarding formatting, I would say that you want either one tab as a separator, so one can use cut or one can set it wide enough to format the output correctly, or you want to indent using spaces, where the result is guaranteed in the case that a monospace font is used.

Moreover, I would consider repeating the print statement to get rid of the trailing `\n`.

Reply:
I think I can simplify a little more these functions.

Reply:
Done.